modules/dnn/src/tflite/tflite_importer.cpp

```diff
@@ -75,11 +79,12 @@ class TFLiteImporter {
 
     void parseFusedActivation(const Operator& op, ActivationFunctionType activ);
     void parseActivation(const Operator& op, const std::string& opcode, LayerParams& layerParams, bool isFused);
-    void addLayer(LayerParams& layerParams, const Operator& op);
-    int addPermuteLayer(const std::vector<int>& order, const std::string& permName, const std::pair<int, int>& inpId, int dtype);
+    void addLayer(LayerParams& layerParams, const Operator& op, bool additionalPreLayer = false, bool additionalPostLayer = false);
```

Comment:
@alexlyulkov may I ask please describe in few words what is the meaning of PreLayer?

Reply:
@alexlyulkov, thanks!

With proposed approach of fused layers naming user has a model with unexpected layers names. What it we keep original fused layer name at the last unfused layer chain but rename only pre-layers?

```patch
If a layer has a fused activation, I add additionalPostLayer and change input and output names this way:
original: conv_relu(conv123, conv123_input, conv123_output)
- new: conv(conv123, conv123_input, conv123_output_additional_post_layer) + relu(conv123_relu, conv1_output_additional_post_layer, conv123_output)
+ new: conv_relu_pre(conv123, conv123_input, conv123_output_additional_post_layer) + conv_relu(conv123_relu, conv1_output_additional_post_layer, conv123_output)


If a layer has additional preprocessing layer, I change input and output names this way:
original: permute_reshape(reshape345, reshape345_input, reshape345_output)
new: permute_reshape_pre(reshape345_permute, reshape345_input, reshape345_input_additional_pre_layer) + permute_reshape(reshape345, reshape345_input_additional_pre_layer, reshape345_output)
```

So the motivation is to have matching output tensors by names between OpenCV and TFLite for origin layers.

Reply:
I've added you naming for the new engine. Btw I've found out that the TFLite models don't have layer names and our parser just uses layer_outputs[0].name